README.md

```diff
@@ -38,7 +38,7 @@ In the meantime, you can build and run Periskop from source:
 
 ```
 docker build --tag periskop .
-docker run -v path/to/config.yaml:/etc/periskop/periskop.yaml -p 8080:8080 periskop
+docker run -e CONFIG_FILE=/etc/periskop/config/<config-file>.yaml -v /path/to/config/yaml/folder:/etc/periskop/config -p 8080:8080 periskop
```

Comment:
I tested again and I confirm that this change is not necessary:

```
➜  periskop git:(dev/docker-compose) docker run -v /Users/riccardociovati/ghq/github.com/soundcloud/periskop/config.dev.yaml:/etc/periskop/periskop.yaml -p 8080:8080 periskop
2020/06/23 12:43:52 Using baseDir /periskop
2020/06/23 12:43:52 Using configFile /etc/periskop/periskop.yaml
2020/06/23 12:43:52 Serving on address 0.0.0.0:8080
2020/06/23 12:43:57 Received new dns resolution result for periskop. Address resolved: 1
2020/06/23 12:43:57 Received new dns resolution result for mock-target. Address resolved: 1
2020/06/23 12:44:02 mock-target: scraped 0 errors from 1 instances
2020/06/23 12:44:02 periskop: scraped 0 errors from 1 instances
....
```

Maybe for you it doesn't work because of mobymac? I'm using Docker for Mac.